### PR TITLE
Version next wizard

### PR DESCRIPTION
**Pull request by Wagnerp**
- `KryptonWizard` control

### CODE REVIEW

File: Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Wizard/Classes/General/ExtensionAttributeFor.NET_2.0.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ * Use of this source code is governed by a BSD-style
+ * license or other governing licenses that can be found in the LICENSE.md file or at
+ * https://raw.githubusercontent.com/Krypton-Suite/Extended-Toolkit/master/LICENSE
+ */
+#endregion
+
+namespace System.Runtime.CompilerServices
+{
+    /// <summary>
+    /// Attribute allowing extenders to be used with .NET Framework 2.0.
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Wizard/Classes/General/ExtensionAttributeFor.NET_2.0.cs]: Net 2.0?? When is this needed when the suite states 3.5 onwards ?

**Comment by Smurf-IV** [comment]
A lot of this code feels like it came out of the ark; i.e. it's from .Net 2.0.
And, it's reinventing a lot of what is already built into Krypton drawing (i.e. buttons / them handling, font handling / etc)
Perhaps time to stand back and re-use Krypton controls, and place into a wizard, rather than takes this and force krypton into it.

After all, it's a control that should just allows forms (usercontrols as Krypton Panels) to be placed into it, with Next - Previous buttons along the bottom ;-)

**Comment by Wagnerp** [comment]
@Smurf-IV Seems to be a cut down version of this: https://github.com/dahall/AeroWizard

**Comment by Smurf-IV** [comment]
> 
> 
> @Smurf-IV Seems to be a cut down version of this: https://github.com/dahall/AeroWizard

Needs "Cutting down" even more. (i.e. restart!)

**Comment by Wagnerp** [comment]
@Smurf-IV I've redone the wizard control.

**Pull request merged**